cmd/vega/init.go

```diff
@@ -146,6 +160,13 @@ func (opts *InitCmd) Execute(args []string) error {
 	return nil
 }
 
+func setTendermintDefaults(cfg *tmcfg.Config) {
```

Comment:
ah, I was sure we were doing it but could not find it...

Reply:
well, where it is, is when the abci gets started cmd/vega/node/node.go:startABCI -> core/blockchain/abci/tm_node.go:NewTmNode -> loadConfig() -> overwriteConfig.

Reply:
yea indeed, I found it @ze97286, my question was malformed. Where do you think we should override these. Where it is actually, which is when starting the node, which also means the user could not change them through the config as we always override them, or when we initialize the config, meaning that we provide the users with defaults we think are right, but they can still change them.

Reply:
well, the purpose of making those overrides was to *not* give them a chance to do what they do because those are things that we rely on. If that's not the case then you can leave it in the init and remove it from the start, or split it so some things are in init and some on start, idk if it makes sense.

Reply:
right.